### PR TITLE
Fixing the project to be Carthage compatible and work with CocoaAsyncSocket

### PR DESCRIPTION
**Pull request by KyleLeneau**
Including the source of another popular library was a code smell to me and did not allow us to work with the library. This change makes the CocoaAsyncSocket dependency external to the project and avoids duplicate symbol issues for other consumers.

### CODE REVIEW

**Comment by joeljfischer** [comment]
Hi @KyleLeneau, thanks for this. Including CAS was never meant to be a permanent solution, but only to test another project. I'm reviewing it now.

File: README.md [hunk 1]
```diff
@@ -20,3 +20,21 @@ Sometimes developers need an embedded HTTP server in their app. Perhaps it's a s
 -   Heavily commented code
 -   Very easily extensible
 -   WebDAV is supported too!
+
+## Getting Started
+
+If you’re using [Carthage](https://github.com/Carthage/Carthage), simply add LivioHTTPServer to your `Cartfile`:
+
+```
+github "PhysicalGraph/LivioHTTPServer"
```
Comment (joeljfischer) [inline README.md]: This should be "livio/LivioHTTPServer"

File: README.md [hunk 2]
```diff
@@ -20,3 +20,21 @@ Sometimes developers need an embedded HTTP server in their app. Perhaps it's a s
...
+```
+Make sure to add `CocoaAsyncSocket.framework` to "Linked Frameworks and Libraries" and "copy-frameworks" Build Phases.
```
Comment (joeljfischer) [inline README.md]: Should include a link to https://github.com/Carthage/Carthage#if-youre-building-for-ios-tvos-or-watchos

**Comment by joeljfischer** [comment]
Other than the noted issues, this looks good!

**Comment by KyleLeneau** [comment]
Thanks @joeljfischer, back to you for review and merge based on those documentation changes.

**Pull request merged**